easybuild/easyconfigs/b/BFAST/BFAST-0.7.0a-goolf-1.4.10.eb

```diff
@@ -24,11 +24,8 @@ description = """BFAST facilitates the fast and accurate mapping of short reads
 
 toolchain = {'name': 'goolf', 'version': '1.4.10'}
 
-import string
-swdir = version.rstrip(string.lowercase + string.uppercase)
-# eg. http://sourceforge.net/projects/bfast/files/bfast/0.7.0/bfast-0.7.0a.tar.gz/download
+source_urls = ['http://sourceforge.net/projects/bfast/files/bfast/%s/' % version.rstrip('a'), 'download']
```

Comment:
An alternative closer to the original which does not involve user intervention:

`swdir = ''.join(char for char in version if not char.isalpha())`
`source_urls = ['http://sourceforge.net/projects/bfast/files/bfast/%s/' % swdir, 'download']`